Mathlib/Algebra/CubicDiscriminant.lean

```diff
@@ -37,7 +37,14 @@ noncomputable section
 /-- The structure representing a cubic polynomial. -/
 @[ext]
 structure Cubic (R : Type*) where
-  (a b c d : R)
+  /-- The cubic's degree-3 coefficient -/
```

Comment:
```suggestion
  /-- The degree-3 coefficient -/
```
since the hover text will already include `self : Cubic`, and this is consistent with the other docstrings in this PR